src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/GerritNotifier.java

```diff
@@ -129,4 +129,21 @@ public void buildCompleted(MemoryImprint memoryImprint, TaskListener listener) {
             logger.error("Could not complete BuildCompleted notification!", ex);
         }
     }
+
+    /**
+     * Helper method which truncates the Gerrit command before it is printed on
+     * the Jenkins log. If the string contains multiple lines, then the first
+     * and last line is retained, all others are discarded.
+     *
+     * @param str String to be truncated
+     * @return Truncated string
+     */
+    private static String truncateCommandForLog(String str) {
+        int firstBreak = str.indexOf('\n');
+        int lastBreak = str.lastIndexOf('\n');
+        if (firstBreak == -1) {
+            return str;
+        }
+        return str.substring(firstBreak) + " ... <truncated> ..." + str.substring(lastBreak, str.length());
```

Comment:
Does this work as intended?
line1
line2
line3
would truncate as:
line2
line3 ... <truncated> ... line3

Reply:
Well this is embarrassing. But let's go with the decreased log level solution instead :)